store/tikv/backoff.go

```diff
@@ -148,7 +148,7 @@ const (
 	getMaxBackoff           = 20000
 	prewriteMaxBackoff      = 20000
 	cleanupMaxBackoff       = 20000
-	GcMaxBackoff            = 100000
+	GcOneRegionMaxBackoff   = 20000
```

Comment:
Why change this value?

Reply:
I don't think gc command can definitely returns in 20 secs.

Reply:
ok, I will change it for one request